android/src/main/kotlin/dev/steenbakker/mobile_scanner/MobileScanner.kt

```diff
@@ -77,7 +81,17 @@ class MobileScanner(
     @ExperimentalGetImage
     val captureOutput = ImageAnalysis.Analyzer { imageProxy -> // YUV_420_888 format
         val mediaImage = imageProxy.image ?: return@Analyzer
-        val inputImage = InputImage.fromMediaImage(mediaImage, imageProxy.imageInfo.rotationDegrees)
+
+        // Inversion
```

Comment:
```suggestion
        // Invert every other frame.
```

The comment next to the assignment can be removed. We can just keep the comment at the top.